tests/parsers/beckman_hiac/beckman_hiac_parser_test.py

```diff
@@ -0,0 +1,49 @@
+from pathlib import Path
+
+import pytest
+
+from allotropy.parser_factory import Vendor
+from allotropy.parsers.beckman_hiac.hiac_parser import HIACParser
+from allotropy.parsers.utils.timestamp_parser import TimestampParser
+from allotropy.to_allotrope import allotrope_from_file
+
+
+@pytest.fixture()
+def test_file() -> Path:
+    f = Path(__file__).parent / "testdata/hiac_example_1.xlsx"
+    return f.absolute()
+
+
```

Comment:
We should have a test in here that checks the contents of the example json file, see the     `validate_schema` and `validate_contents` functions in other parser tests.

Reply:
For `validate_contents`, given that we have `NaN` in our outputs, I think we need to use `ignore_nan_inequality=True` in `DeepDiff`. In our example. We get the following.

As is:

```
DeepDiff(
    expected_dict,
    allotrope_dict,
    exclude_regex_paths=exclude_regex,
    ignore_type_in_groups=[(float, np.float64)],
)
```

Results:
```
{'values_changed': {"root['measurement document']['distribution document'][0]['distribution'][0]['differential particle density']['value']": {'new_value': nan,
   'old_value': nan},
  "root['measurement document']['distribution document'][0]['distribution'][0]['differential count']['value']": {'new_value': nan,
   'old_value': nan},
  "root['measurement document']['distribution document'][0]['distribution'][1]['differential particle density']['value']": {'new_value': nan,
   'old_value': nan},
  "root['measurement document']['distribution document'][0]['distribution'][1]['differential count']['value']": {'new_value': nan,
   'old_value': nan},
  "root['measurement document']['distribution document'][0]['distribution'][2]['differential particle density']['value']": {'new_value': nan,
   'old_value': nan},
  "root['measurement document']['distribution document'][0]['distribution'][2]['differential count']['value']": {'new_value': nan,
   'old_value': nan},
  "root['measurement document']['distribution document'][0]['distribution'][3]['differential particle density']['value']": {'new_value': nan,
   'old_value': nan},
  "root['measurement document']['distribution document'][0]['distribution'][3]['differential count']['value']": {'new_value': nan,
   'old_value': nan},
  "root['measurement document']['distribution document'][0]['distribution'][4]['differential particle density']['value']": {'new_value': nan,
   'old_value': nan},
  "root['measurement document']['distribution document'][0]['distribution'][4]['differential count']['value']": {'new_value': nan,
   'old_value': nan}}}

```

However, if we add that param to `DeepDiff`, we can pass

```
DeepDiff(
    expected_dict,
    allotrope_dict,
    exclude_regex_paths=exclude_regex,
    ignore_type_in_groups=[(float, np.float64)],
)
```

Results:
```
{}
```

Thoughts @nathan-stender?

Reply:
`ignore_nan_inequality=True` seems like a fine adjustment to me.

 However, I'm kind of surprised `NaN` passes schema validation. I'm no expert, but it seems like most things I look at say that `NaN` is not valid JSON. It seems like this is a bug in the validator we use (https://github.com/tdegrunt/jsonschema/issues/53)

I would say we should just omit fields when their value is `NaN`, but it seems like at least one of these (`differential count`) is required in the schema. This is possibly a question for @LimulusPolyphemus

Reply:
As discussed, I've updated the model to make these two missing fields optional. We will regroup with Allotrope as we work to enhance this model.